crates/torin/src/values/size.rs

```diff
@@ -166,7 +166,10 @@ impl Scaled for Size {
         match self {
             Size::Pixels(s) => *s *= scale_factor,
             Size::DynamicCalculations(calcs) => {
-                calcs.iter_mut().for_each(|calc| calc.scale(scale_factor));
+                calcs.insert(0, DynamicCalculation::OpenParenthesis);
+                calcs.push(DynamicCalculation::ClosedParenthesis);
+                // implicit multiplication
```

Comment:
Lets make it no implicit for the sake of not under-simplifying

Reply:
Done :D